server/main.example.go

```diff
@@ -0,0 +1,26 @@
+package server
```

Comment:
@arlolra @masomel : What do you think about fd73f1ccca5c29fcc6a93e55cabe097dd5fc5778  and 78db200a2a10b46ece5c5a973d384e3bbe330921?